test/functionalTest/harnessFunctions.sh

```diff
@@ -153,7 +198,22 @@ function dbDrop()
   fi
 }
 
+# ------------------------------------------------------------------------------
+#
+# dbDropTenant - 
+#
+function dbDropTenant()
+{
+  db=$1
```

Comment:
Fixed in 3ef4ea3 and bd93c95, dbInitTenant and dbDrop has been removed and the code included in the function.